tests/publiccloud/run_ltp.pm

```diff
@@ -85,9 +95,12 @@ sub run {
     my ($self, $args) = @_;
     my $arch = check_var('PUBLIC_CLOUD_ARCH', 'arm64') ? 'aarch64' : 'x86_64';
     my $ltp_repo = get_var('LTP_REPO', 'https://download.opensuse.org/repositories/benchmark:/ltp:/stable/' . generate_version("_") . '/');
+    
     my $provider;
     my $instance;
 
+    record_info('LTP RUNTIME', $ltp_runtime);
```

Comment:
The `LTP_RUNTIME` setting can be directly seen in the SETTINGS tab, I would say that this is not really adding some new information anywhere.

Reply:
It is to help the log analysis without switching context and it has been useful to check the correct application of the new  parameter keeping the attention on the execution log, marked with the expected runtime.